src/DebugBar/Resources/widgets/sqlqueries/widget.js

```diff
@@ -14,15 +14,7 @@
 
         onFilterClick: function(el) {
             $(el).toggleClass(csscls('excluded'));
-
-            var excludedLabels = [];
-            this.$toolbar.find(csscls('.filter') + csscls('.excluded')).each(function() {
-                excludedLabels.push(this.rel);
-            });
-
```

Comment:
I did remove this code because it does nothing, seems like a copy/paste issue 
https://github.com/maximebf/php-debugbar/blob/1fdc3a3007561721daec958a1a75d7a062d1e6ab/src/DebugBar/Resources/widgets.js#L438-L449